### ISSUE TITLE
my closed pull requests

### ISSUE THREAD

**Issue by adrelanos** [grml/grml-debootstrap#253]
Sorry about the closed pull requests noise.

I was attempting to make github actions work in my personal (soft) fork (only for the purpose of contributing and carrying delta until merged here upstream). Previously I had pressed the github fork button. But apparently such repositories cannot run github actions.

To have a "real" fork I deleted the repository, manually created and pushed. That fixed the github actions in my personal repository but had the unwanted side effects of closing all my prior pull requests here upstream. I am trying to figure out how to re-open these.

**Comment by adrelanos** [comment]
Here's a list:

* https://github.com/grml/grml-debootstrap/pull/234
* https://github.com/grml/grml-debootstrap/pull/231
* https://github.com/grml/grml-debootstrap/pull/248
* https://github.com/grml/grml-debootstrap/pull/123
* https://github.com/grml/grml-debootstrap/pull/124
* https://github.com/grml/grml-debootstrap/pull/236

Seems like these cannot be re-opened. I'll create new PRs for the recent / realistic ones.

**Comment by mika** [comment]
Yeah, this was quite some noise in my inbox :smiling_face_with_tear: 

> Seems like these cannot be re-opened. I'll create new PRs for the recent / realistic ones.

ACK, please do so about the issues your care about, closing this one now
